sessionctx/variable/varsutil_test.go

```diff
@@ -605,17 +601,22 @@ func (s *testVarsutilSuite) TestValidate(c *C) {
 		{TiDBAllowFallbackToTiKV, "tiflash,tikv,tidb", true},
 	}
 
-	for _, t := range tests {
-		_, err := GetSysVar(t.key).Validate(v, t.value, ScopeGlobal)
-		if t.error {
-			c.Assert(err, NotNil, Commentf("%v got err=%v", t, err))
-		} else {
-			c.Assert(err, IsNil, Commentf("%v got err=%v", t, err))
-		}
+	for _, tc := range testCases {
+		t.Run(tc.key, func(t *testing.T) {
+			t.Parallel()
+			_, err := GetSysVar(tc.key).Validate(v, tc.value, ScopeGlobal)
+			if tc.error {
+				require.Error(t, err)
```

Comment:
We have a `require.Errorf` function and also `NoErrorf`.

Reply:
Fixed it with the following commit - 9f0a372c9660f842effd6baa9b939998914db125

Reply:
Thanks!